### PR TITLE
Source install instructions and ROS 2 folder restructure

### PR DESCRIPTION
**Pull request by roni-kreinin**
- ROS 2 contents are now directly under docs/ros/
- Added static folder under docs/ for ROS 2 specific assets

### CODE REVIEW

**Review by tonybaltovski — APPROVED** [review]

File: docs/ros/installation/robot.mdx [hunk 1]
```diff
@@ -56,6 +56,35 @@ sudo apt install ros-humble-clearpath-robot
...
+```
+
+**3. Build packages**
+
+```
+colcon build --symlink-install
+```
+
+:::note
+Make sure to add the workspace to your `robot.yaml` file. TODO: Add link to robot yaml documentation
```
Comment (jhiggins-cpr) [inline docs/ros/installation/robot.mdx:85]: 1. I think there is supposed to be a blank line before and after each ":::" line.
2. Just noting that there is still a "TODO" here.

**Review by jhiggins-cpr — APPROVED** [review]
Looks good to me. Just a couple minor notes.

**Comment by hilary-luo** [comment]
Since we are treating the ros2-humble assets as versioned assets they need to be addressed using relative paths. See https://docusaurus.io/docs/versioning#global-or-versioned-collocated-assets . I think that this also means that their folders shouldn't get called out as static directories in the config because that will place them into the root of the build directory and may cause conflicts across versions.

**Comment by roni-kreinin** [comment]
@hilary-luo I tried just placing them in a regular folder but the images wouldn't show up. Is there something else I have to do for relative paths to work?

**Comment by hilary-luo** [comment]
It requires either using the markdown method of inserting an image (``) or a slight modification to the image src line using the nodejs functionality of `require` and `default`. For example in the ros -> tutorials -> simulator -> overview.mdx, if we have the simulation.png file in the same folder as the mdx file, you can insert it like this:

```
<center>
  <figure>
    <img
      src={require('./simulation.png').default}
      width="800"
    />
    <figcaption>Clearpath Simulator</figcaption>
  </figure>
</center>

**Review by hilary-luo — APPROVED** [review]

**Pull request merged**